frontend/src/components/ui-kit/components/Input/InputSearch/InputSearch.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, memo } from 'react';
+import styles from './InputSearch.module.scss';
+
+import { ICONS } from '../../../icons';
+import IconButton from '../../IconButton';
+
+const InputSearch = memo(({ search, setSearch }) => {
+  const [smallScreen, setSmallScreen] = useState(false);
+  const [isFocused, setIsFocused] = useState(false);
+
+  useEffect(() => {
+    const mediaQuery = window.matchMedia('(max-width: 375px)');
+
+    const handleMediaChange = (event) => {
+      setSmallScreen(event.matches);
+    };
+    mediaQuery.addListener(handleMediaChange);
+    handleMediaChange(mediaQuery);
+
+    return () => {
+      mediaQuery.removeListener(handleMediaChange);
+    };
+  }, []);
+
+  return (
+    <div className={styles.root}>
+      {smallScreen ? (
+        <IconButton icon={<ICONS.back />} />
+      ) : isFocused ? (
+        <ICONS.searchGreen />
+      ) : (
+        <ICONS.search />
+      )}
+      {search && (
+        <div className={styles.clearBtn}>
+          <IconButton
+            onClick={() => setSearch('')}
```

Comment:
pass the function for cleaning the inputs to the prop, maybe it will do something